### ISSUE TITLE
Project History not available in the menu when opening an empty new window

### ISSUE THREAD

**Issue by mh-nichts** [atom/atom#13948]
### Description

When opening a new window, with no folder or project in the tree view, the project history is not available in the menu File > Reopen Project.

### Steps to Reproduce

1. While Atom is open, go to File > New window.
2. In the new window, go to File > Reopen Project.
3. Notice there is no project listed under "Clear Project History".
4. So instead, go to File >Open folder... and open any folder (it works also with "Add a project folder..." or  "Open file...").
5. Now go again to File > Reopen Project.
6. You now see the list of last projects under "Clear Project History".

**Expected behavior:**
On step 3 : under "Clear Project History", I expect to see the list of the last projects I worked on, the same as on step 6.

**Actual behavior:**
There is nothing under "Clear Project History" and the separator.
The list is available only after opening any folder or file.

**Reproduces how often:**
Everytime I open an empty new window, whether I already have Atom open or not.

I first thought Atom needed time to retrieve the list, but I tried and left an empty window open for a long time (several minutes) and the list was still not available.

### Versions

Atom    : 1.14.4
Electron: 1.3.13
Chrome  : 52.0.2743.82
Node    : 6.5.0

I'm running Atom on Windows 10 (64 bits).

A colleague of mine using Atom confirms the same behavior (also on Windows 10).

**Comment by Ben3eeE** [comment]
Thanks for taking the time to contribute!

We noticed that this is a duplicate of https://github.com/atom/atom/issues/13758.

Because we treat our issues list as [the Atom team's backlog](https://en.wikipedia.org/wiki/Scrum_(software_development)#Product_backlog), we close duplicates to focus our work and not have to touch the same chunk of code for the same reason multiple times. This is also why we may mark something as duplicate that isn't an exact duplicate but is closely related.

For information on how to use GitHub's search feature to find out if something is a duplicate before filing, see [the How Can I Contribute? section of the Atom CONTRIBUTING guide](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-can-i-contribute).

**Comment by mh-nichts** [comment]
Thank you, sorry that I didn't find this other issue before !
